### PR TITLE
Removing references to "KycoCoreNApps"

### LINKED ISSUE #232: Remove references to "KycoCoreNApp"
All NApps now have the same "status", there are no more "CoreNApps".

### PR DESCRIPTION
**Pull request by diraol**
FIX #232

### CODE REVIEW

File: kyco/core/napps.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 log = logging.getLogger(__name__)
 
-__all__ = ('KycoCoreNApp', 'KycoNApp')
+__all__ = ('KycoNApp')
```
Comment (cemsbr) [inline kyco/core/napps.py:10]: A valid tuple here would be `__all__ = ('KycoNApp',)` or simply `__all__ = 'KycoNApp',` (always with comma).

**Review by cemsbr — CHANGES_REQUESTED** [review]
We also depend on removing KycoCoreNApp references in kyco-core-napps for the tests to pass.

File: kyco/core/napps.py (hunk 1 above)
Comment (diraol) [reply kyco/core/napps.py:10]: @cemsbr, the KycoCoreNapps parte is on another PR. :)

**Comment by cemsbr** [comment]
Dependency: kytos/kyco-core-napps#170

File: kyco/core/napps.py (hunk 1 above)
Comment (cemsbr) [reply kyco/core/napps.py:10]: I noticed later and made a reference.

**Pull request merged**